### DISCUSSION TITLE
ValueError - No Device Langid (permission issue, no string descriptors supported or device error)

### DISCUSSION THREAD

**Discussion by JLittleGDS96** [pyusb/pyusb#417] [Q&A] [answered]
Hello! Firstly, I am quite new to this so I am still learning, hope you guys don't mind if this has been asked. I have did some search and it did not really help me.

Apparently, my friend was able to execute the same code on the same device properly. So I am assuming the device error is out. Not sure if it's the issue with my laptop permissions. 

Thank you for reading! 

I am using the Windows 10, trying to execute python programmes to the Lego EV3 device via USB.

![tempsnip](https://user-images.githubusercontent.com/101244201/158562412-2b4d8b34-7a27-4c4a-8597-3e519d366799.png)

**Comment by jonasmalacofilho** [comment]
I'm not aware of permissions issues on Windows.

Did your friend use the exact same device, or just another device of the same model?  And have you adjusted the `host` parameter in `ev3.EV3()` to your specific device unit?

**Comment by mcuee** [comment, marked as answer]
Have you installed the right device driver (libusb Windows compatible)?
